### ISSUE TITLE
PicoDet training on Custom Layout Dataset

### ISSUE THREAD

**Issue by bely66** [PaddlePaddle/PaddleOCR#8697]
Hi Everyone,
I'm training the layout model using a custom dataset for Layout with only Tables and Text.
I downloaded the CDLA model as a pretrained model to fine-tune on my data.
The dataset has 1200 document in a non-latin and non-chinese language.
The model didn't cross 20% accuracy till now.
What might be the issue?
Do I need to train for longer number of epcochs I set it to 100 initially?
Do I need to add more data?
What can be done to help the model generalize?

**Comment by an1018** [comment]
You can try use the pretrained model of PubLayNet
